dacsspace/reporter.py

```diff
@@ -23,6 +23,7 @@ def write_report(self, results, invalid_only=True):
             fieldnames = [
                 "uri",
                 "valid",
+                "error count",
```

Comment:
I am wondering what the implications are, if any, of having a space in a dictionary key/CSV column name? Anyone have thoughts about this?